terraform/variables.tf

```diff
@@ -12,24 +12,15 @@ variable paas_web_app_instances { default = 1 }
 
 variable paas_web_app_memory { default = 512 }
 
-variable paas_rails_env {}
-
-variable paas_sentry_dsn {}
-
-variable paas_secret_key_base {}
-
-variable paas_settings_cycle_ending_soon {}
-
-variable paas_settings_cycle_has_ended {}
-
-variable paas_settings_display_apply_button {}
-
-variable paas_settings_google_gcp_api_key {}
+variable paas_logstash_url {}
 
-variable paas_settings_google_maps_api_key {}
+variable paas_app_config_file { default = "./workspace_variables/app_config.yml" }
 
-variable paas_logstash_url {}
+variable paas_app_secrets_file { default = "./app_secrets.yml" }
 
 locals {
-  cf_api_url = "https://api.london.cloud.service.gov.uk"
+  cf_api_url                     = "https://api.london.cloud.service.gov.uk"
+  app_config                     = yamldecode(file(var.paas_app_config_file))
```

Comment:
Isn't it `yamldecode(file(var.paas_app_config_file))[var.paas_app_environment]` ?

Reply:
yes, I must have reversed the two vars, fixed it now. thanks